benchmarks/folds.hs

```diff
@@ -31,6 +31,7 @@ main = defaultMainWith config
       [ bench "native"     $ nf (V.toList . V.indexed) v
       , bench "itraversed" $ nf (itoListOf itraversed) v
       ]
+    , bench "sum" $ whnf (sumOf each) v
```

Comment:
For consistency with the other benchmarks, can you turn this into a `bgroup "sum"` that also includes benchmarks for `whnf sum v`? (And similarly for the other `sumOf` benchmarks.)